### ISSUE TITLE
Cannot read property 'debug' of undefined

### ISSUE THREAD

**Issue by ealves-pt** [jsreport/jsreport-weasyprint-pdf#1]
Hello since updating to jsreport@2.2.0 I've been facing the following error:
```TypeError: Cannot read property 'debug' of undefined
    at /home/ea/jsreport/node_modules/jsreport-weasyprint-pdf/lib/weasyprint.js:34:17
    at Array.forEach (<anonymous>)
    at /home/ea/jsreport/node_modules/jsreport-weasyprint-pdf/lib/weasyprint.js:33:16
    at tryCatcher (/home/ea/jsreport/node_modules/jsreport-weasyprint-pdf/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/home/ea/jsreport/node_modules/jsreport-weasyprint-pdf/node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (/home/ea/jsreport/node_modules/jsreport-weasyprint-pdf/node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (/home/ea/jsreport/node_modules/jsreport-weasyprint-pdf/node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (/home/ea/jsreport/node_modules/jsreport-weasyprint-pdf/node_modules/bluebird/js/release/promise.js:693:18)
    at Promise._fulfill (/home/ea/jsreport/node_modules/jsreport-weasyprint-pdf/node_modules/bluebird/js/release/promise.js:638:18)
    at /home/ea/jsreport/node_modules/jsreport-weasyprint-pdf/node_modules/bluebird/js/release/nodeback.js:42:21
    at /home/ea/jsreport/node_modules/phantom-workers/lib/phantomManager.js:141:9
    at IncomingMessage.<anonymous> (/home/ea/jsreport/node_modules/phantom-workers/lib/phantomWorker.js:178:13)
    at emitNone (events.js:111:20)
    at IncomingMessage.emit (events.js:208:7)
    at endReadableNT (_stream_readable.js:1064:12)
    at _combinedTickCallback (internal/process/next_tick.js:138:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
```

Any idea on what this is?

**Comment by bjrmatos** [comment]
> Any idea on what this is?

yes, this is because the `weasyprint-pdf` does not support the latest release of jsreport (2.2.0), you need to go back to the previous version you were using until we update weasyprint-pdf for latest jsreport compatibility

**Comment by ealves-pt** [comment]
Is there any estimation on releasing that?

I will try to rollback to the previous version of jsreport but I'm afraid of that process.

Do you know if there is any table somewhere where I can see for each extension the version that is valid for **jsreport@1.10.0**?

**Comment by bjrmatos** [comment]
> Is there any estimation on releasing that?

we are about to release jsreport `2.3.0`, our priority is doing that, which should be tomorrow or in the week, after that we will check compatibility of extensions and prepare the releases for them. so i guess it can take us one or some weeks to check this.



> Do you know if there is any table somewhere where I can see for each extension the version that is valid for **jsreport@1.10.0**?

unfortunately no, but as a general rule (which most extensions follow) is extensions that have 2.x.x version means that it is compatible with jsreport 2.x.x, if it is version is 1.x.x it is compatible with jsreport 1.x.x, there are some exceptions to this rule (like this extension) which does not follow this match

**Comment by ealves-pt** [comment]
Thank you very much for the reply.

I decided in the meantime to not rollback to version 1.x of jsreport and, for the meantime, I'll be using chrome-pdf and I will survive without the css `@page` rule for a bit.

**Comment by bjrmatos** [comment]
@ealves-pt so i just released `jsreport-weasyprint-pdf` version `1.0.0` which is compatible with latest version of jsreport (2.x.x), hope it works now for you

**Comment by ealves-pt** [comment]
Works perfectly!!! Thank you very much! :+1: :+1: :muscle: :muscle:
